### PR TITLE
I think this is what prof wants

### PR DESCRIPTION
**Pull request by krastegar**
I built dockerfile and dockercompose. All of the calculations are done inside of the bash script. The results are outputted in the root directory in a file called results.csv

### CODE REVIEW

File: database_import.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+DATABASE_TO_COPY_INTO="baseball"
+DATABASE_FILE="baseball.sql"
+PASS="ROOT_ACCESS_PASSWORD"
+
+mysql -u root -p${PASS} -h mariadb -e "CREATE DATABASE IF NOT EXISTS ${DATABASE_TO_COPY_INTO}"
+mysql -u root -p${PASS} -h mariadb ${DATABASE_TO_COPY_INTO} < ${DATABASE_FILE}
+echo "DATABASE IMPORT DONE....."
+echo "Starting HW......."
+
+mysql -u root -p${PASS} -h mariadb baseball < src/SQL_BAIN_OF_MY_EXISTENCE.sql > results.csv
```
Comment (Zack1803) [inline database_import.sh]: I do not see your results.csv file...maybe you want to check on that.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+version: "3.3"
```
Comment (Zack1803) [inline docker-compose.yml:1]: yml file looks good

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      context: .
+      dockerfile: Dockerfile
+    image: baseball
+    ports:
+      - "5001:8080"
+    environment:
+      PORT: "8080"
+    restart: unless-stopped
+    volumes:
+      - "/home/bioinfo/Desktop/baseball.sql:/app/baseball.sql"
```
Comment (Zack1803) [inline docker-compose.yml]: Are you able to mount the baseball.sql using this command?

File: src/SQL_BAIN_OF_MY_EXISTENCE.sql [hunk 4]
```diff
@@ -19,12 +19,12 @@ ADD CONSTRAINT PRIMARY KEY(ID);
 DROP TABLE IF EXISTS Hist_Avg;
```
Comment (Zack1803) [inline src/SQL_BAIN_OF_MY_EXISTENCE.sql:20]: SQL file looks good. Logic seems to be correct and the query is quick and efficient.

File: requirements.txt [hunk 5]
```diff
@@ -5,35 +5,41 @@
 #    pip-compile --output-file=requirements.txt requirements.in
 #
 contourpy==1.0.6
-    # via matplotlib
+    # via
+    #   -r requirements.in
```
Comment (Zack1803) [inline requirements.txt:9]: I am unsure as to why do you need requirements.in file for this assignment..I am assuming you are preparing this script for Final Project :)

File: database_import.sh (hunk 1 above)
Comment (krastegar) [reply database_import.sh]: Hi Meghan, once you run the container and receive the message "Done....", you have to make a new terminal and type the command 
`docker exec -it bda_602_SQL_ROL_AVG_1 /bin/bash` 
This should take you to the home directory of the container and there you can find results.csv

**Pull request merged**